genetic_algorithms/algorithm_wrappers/plot_cost_function_wrapper.py

```diff
@@ -0,0 +1,9 @@
+from genetic_algorithms.problems.base.state import State
+from genetic_algorithms.problems.base.model import Model
+from genetic_algorithms.algorithm_wrappers.algorithm_wrapper import AlgorithmWrapper
+
+
+class CostPrinterWrapper(AlgorithmWrapper):
+    def _perform_side_effects(self, model: Model, state: State):
+        new_cost = model.cost_for(state)
+        print('New cost received: ', new_cost)
```

Comment:
Tylko jak POC. 
Chciałem zrobić tak żeby rysowały się wykresy funkcji kosztu, ale okazało się, że to jest trudnieszje niż się wydaje, więc się poddałem i zrobiłem zwykle printowanie :D